src/app/shared/footer/footer.html

```diff
@@ -5,19 +5,23 @@
       <div class="footer-logo">
         <img class="docs-footer-angular-logo"
           src="../../../assets/img/homepage/angular-white-transparent.svg"
-          alt="angular">
+          alt="Angular icon">
```

Comment:
Shouldn't it be "Angular logo"?

Reply:
Not based on https://hudsonvalleygraphics.com/is-it-a-logo-or-an-icon/ which aligns with my thinking that most logos include some typography.

https://blog.designcrowd.com/article/353/differences-between-logos-and-icons also mentions that icons are generally quadratic/square.

Here's the Angular logo: https://angular.io/assets/images/logos/angular/logo-nav@2x.png